mobile-v3/lib/src/meta/constants/aqi_ranges.dart

```diff
@@ -0,0 +1,10 @@
+ import '../../app/dashboard/models/airquality_response.dart';
+
+final aqiRanges = AqiRanges(
```

Comment:
Why are we still "hard coding" these ranges? Aren't we tired of endless drama? @jordanopos

Reply:
According to the new implementation of the getAirQualityIcon, the function requires an AqiRanges object, and a value. So unless we can have another api to explicitly return the ranges, this is what works best for now.

Reply:
Hi @jordanopos , this need was already addressed on all endpoints. We are explicitly returning the ranges for the API, please check out the attached screenshot.
![Uploading Screenshot 2024-12-15 at 08.25.48.png…]()

Reply:
@Baalmart @jordanopos i had rewritten the function to return the ranges dynamically. But we are using that same function in the forecast response as well. So the change had to be made there as well. The problem causing the forecastloading error is because i dont think the forecast api does not have aqi ranges/measurements.